File: BBIS.Api/ClientApp/src/models/DonorScreening/DonorRecentDonationDto.ts
export interface IDonorRecentDonationDto {
    DonorRecentDonationId: Guid | null;
    NumberOfDonation: number | null;
    DateOfRecentDonation: Date | null;
    PlaceOfRecentDonation: string;
    Agency: string;
}
  
export class DonorRecentDonationDto implements IDonorRecentDonationDto {
    DonorRecentDonationId: Guid | null = '';
    NumberOfDonation: number | null = null;
    DateOfRecentDonation: Date | null = null;
    PlaceOfRecentDonation: string = "";
    Agency: string = "";
}

File: BBIS.Application/DTOs/DonorScreening/DonorRecentDonationDto.cs
namespace BBIS.Application.DTOs.DonorScreening
{
    public class DonorRecentDonationDto
    {
        public Guid? DonorRecentDonationId { get; set; }
        public Guid DonorId { get; set; }
        public int NumberOfDonation { get; set; }
        public DateTime DateOfRecentDonation { get; set; }
        public string PlaceOfRecentDonation { get; set; }
        public string Agency { get; set; }
    }
}
